hubgrep/frontend_blueprint/templates/components/collapsible/_collapsible.scss

```diff
@@ -48,13 +48,12 @@
   }
 
   .collapsible-content {
-    max-height: 0;
+    display: none;
     overflow: hidden;
-    transition: max-height .25s ease-in-out;
   }
 
   .toggle:checked + .lbl-toggle + .collapsible-content {
-    max-height: 700px; // we need a specific height for transition to take effect
+    display: unset;
```

Comment:
To completely fix this problem with a cutoff, I simply can't have a transition effect as it requires a specific max value (so we increase it, and later on its broken again when theres too much in it). So byebye transition effect.

Reply:
To add to this, I WANT to have a barely visible transition effect for something rolling down almost instantly as it gives softer effect on displacing whatever other elements are on the page, as well as the softer roll-out of the collapsibles content.